bufferflow.go

```diff
@@ -19,6 +19,7 @@ type Bufferflow interface {
 	Init()
 	BlockUntilReady(cmd string, id string) (bool, bool) // implement this method
 	//JustQueue(cmd string, id string) bool                     // implement this method
+	OnIncomingDataBinary(data []byte)
```

Comment:
You don't need this new function, you can easily convert `data` from `string` to `[]byte` in `OnIncomingData`.